recipes/pyloh/meta.yaml

```diff
@@ -1,14 +1,19 @@
+{% set name = "pyloh" %}
+{% set version = "1.4.3" %}
+{% set hash = "30a97d37955546bad7e605899b068750e79f0a26583a3c8cb8ec8e4778dc5bc2" %}
+
 package:
-  name: pyloh
-  version: '1.4.1'
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  fn: pyloh-1.4.1.tar.gz
-  url: https://github.com/uci-cbcl/PyLOH/archive/1.4.1.tar.gz
+  fn: pyloh-{{ version }}.tar.gz
+  url: https://github.com/uci-cbcl/PyLOH/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
 
 build:
-  number: 1
-  skip: True # [osx]
+  number: 0
+  skip: True  # [not py27]
 
 requirements:
   build:
```

Comment:
`README.md` says we have the following version requirements:
```yaml
    - numpy >=1.6.1
    - scipy >=0.10
    - pysam >=0.7
    - matplotlib >=1.2
```
Also `setuptools` can be removed from the `build` requirements.